cfgov/unprocessed/css/forms.less

```diff
@@ -606,29 +608,46 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
-.form-l_col-1-2 {
-    .respond-to-min(1060px, {
-        .grid_column(6);
-    });
+.form-label-header + .form-l .form-label-header {
+    // Override heading-5() settings.
+    .webfont-regular();
+    text-transform: none;
+    margin-bottom: unit(10px / @font-size, em);
 }
 
-.form-l_col-1-3 {
-    .respond-to-min(1060px, {
-        .grid_column(4);
+.form-l {
+    .respond-to-min(@bp-sm-min, {
+        .grid_nested-col-group();
     });
 }
 
-.form-l_col-2-3 {
-    .respond-to-min(1060px, {
-        .grid_column(8);
-    });
+.form-l_col input,
+.form-l_col select {
+    box-sizing: border-box;
+    width: 100%;
 }
 
-.form-l_col-1 {
-    .respond-to-min(1060px, {
+.respond-to-min(@bp-sm-min, {
+    .form-l_col-1-4 {
+        .grid_column(1, 4);
```

Comment:
It could, but `1, 4` seems clearer to me since the class name is `form-l_col-1-4`.